catalyst-toolbox/src/bin/cli/rewards/full/config.rs

```diff
@@ -0,0 +1,76 @@
+use std::path::PathBuf;
```

Comment:
are votes rewards in here?

Reply:
Yep, there's a few places (e.g. `vote_count_path` on line 19)